src/main/java/com/ericsson/ei/config/ConfigurationLogger.java

```diff
@@ -1,78 +1,71 @@
-                + "waitlist.fixedRateResend: " + environment.getProperty("waitlist.fixedRateResend") + "\n"
-                + "subscription.collection.name: " + environment.getProperty("subscription.collection.name") + "\n"
-                + "subscription.collection.repeatFlagHandlerName: " + environment.getProperty("subscription.collection.repeatFlagHandlerName") + "\n"
-                + "testaggregated.enabled: " + environment.getProperty("testaggregated.enabled") + "\n"
-                + "threads.corePoolSize: " + environment.getProperty("threads.corePoolSize") + "\n"
-                + "threads.queueCapacity: " + environment.getProperty("threads.queueCapacity") + "\n"
-                + "threads.maxPoolSize: " + environment.getProperty("threads.maxPoolSize") + "\n"
-                + "failed.notification.collection-name: " + environment.getProperty("failed.notification.collection-name") + "\n"
-                + "email.sender: " + environment.getProperty("email.sender") + "\n"
-                + "email.subject: " + environment.getProperty("email.subject") + "\n"
-                + "notification.failAttempt: " + environment.getProperty("notification.failAttempt") + "\n"
-                + "notification.ttl.value: " + environment.getProperty("notification.ttl.value") + "\n"
-                + "spring.mail.host: " + environment.getProperty("spring.mail.host") + "\n"
-                + "spring.mail.port: " + environment.getProperty("spring.mail.port") + "\n"
-                + "spring.mail.username: " + environment.getProperty("spring.mail.username") + "\n"
-                + "spring.mail.properties.mail.smtp.auth: " + environment.getProperty("spring.mail.properties.mail.smtp.auth") + "\n"
-                + "spring.mail.properties.mail.smtp.starttls.enable: " + environment.getProperty("spring.mail.properties.mail.smtp.starttls.enable") + "\n"
-                + "er.url: " + environment.getProperty("er.url") + "\n"
-                + "ldap.enabled: " + environment.getProperty("ldap.enabled") + "\n"
-                + "ldap.server.list: " + environment.getProperty("ldap.server.list") + "\n"
-                + "logging.level.root: " + environment.getProperty("logging.level.root") + "\n"
-                + "logging.level.org.springframework.web: " + environment.getProperty("logging.level.org.springframework.web") + "\n"
-                + "logging.level.com.ericsson.ei: " + environment.getProperty("logging.level.com.ericsson.ei") + "\n"
-                + "\nThese properties are only some of the configurations, more configurations could have been provided.\n");
+        List<MapPropertySource> propertySources = findApplicationProperties();
+        for (MapPropertySource propertySource : propertySources) {
+            String propertiesLogMessage = createPropertiesLogMessage(propertySource);
+            LOGGER.info(propertiesLogMessage);
```

Comment:
Do we want them all to be logged on INFO? And right now it looks like they will be logged separately - does that mean they will be spread out in the console?

Reply:
Ah this is also a mistake. It's supposed to be debug. If you look in createPropertiesLogMessage method I actually compile the complete String with all the properties in one go to prevent what you just mentioned. Otherwise it looks very cluttered with the mandatory logger lines (date, time, class, line etc.)